### DISCUSSION TITLE
Can I dispatch an event in admin panel and listen for it in another panel

### DISCUSSION THREAD

**Discussion by alkoumi** [filamentphp/filament#9130] [Help]
I have two Panels `AdminPanelProvider` with authentication and `MainPanelProvider` for unauthenticated users to show the number of total users using StatsOverview widgets

> I want to dispatch an event when any User created in the AdminPanel

> and listen to that event in the  MainPanel 

I added this to  CreateUser page to dispatch the event 
```
    protected function afterCreate(): void
    {
        $this->dispatch('user-created')->to(Main::class);
    }
```

and added this code to the `MainPanelProvider`
```
->renderHook('panels::footer', fn (): View => view('filament.hooks.footer'))
```

in the footer, I added this  

```
<script>
    document.addEventListener('DOMContentLoaded', () => {
        window.addEventListener('user-created', () => {
            console.log('user-created');
        });
    })
</script>
```

it works only in the same component view

But did not work in the other panel MainPanel

>How Can I dispatch an event in the admin panel and listen for it in another panel?

**Comment by leandrocfe** [comment]
Sorry, I didn't understand your question..

**Reply by alkoumi** [reply]
How Can I dispatch an event in the admin panel and listen for it in another panel?

**Comment by alkoumi** [comment]
I will try the WebSocket and echo
